Mathlib/Topology/MetricSpace/Thickening.lean

```diff
@@ -362,6 +379,9 @@ theorem thickening_iUnion (δ : ℝ) (f : ι → Set α) :
   simp_rw [thickening, infEdist_iUnion, iInf_lt_iff, setOf_exists]
 #align metric.thickening_Union Metric.thickening_iUnion
 
+lemma thickening_biUnion {ι : Type*} (δ : ℝ) (f : ι → Set α) (I : Set ι) :
```

Comment:
This is another proposed but not strictly necessary API lemma about `biUnion`s. (See comments at `iUnion_biUnion`.)